tools/releasable/Akefile

```diff
@@ -22,11 +22,6 @@ use File::Directory::Tree;
 
```

Comment:
↑ The table above still talks about “Local” and “Server” and I think it no longer makes sense.

Reply:
Hmmmm. It is pretty and I am not sure I want to remove it... Though yes, now it makes no sense.